c10/core/Backend.h

```diff
@@ -73,6 +73,10 @@ static inline Backend tensorTypeIdToBackend(TensorTypeId t) {
     return Backend::SparseCUDA;
   } else if (t == SparseHIPTensorId()) {
     return Backend::SparseHIP;
+  } else if (t == QCPUTensorId()) {
+    return Backend::CPU;
```

Comment:
You should make a new backend for quantized tensors, because they will get different kernel implementations.